src/core/models/outputs/outputs.h

```diff
@@ -6,111 +6,121 @@
 
 #include <stdint.h>
 
-#include "core/address.h"
 #include "core/types.h"
-#include "uthash.h"
-
-// Serialized bytes = output type(uint8_t) + address type(uint8_t) + ed25519 address(32bytes) + amount(uint64_t)
-#define UTXO_OUTPUT_SERIALIZED_BYTES (1 + 1 + ADDRESS_ED25519_BYTES + 8)
 
 /**
- * @brief Output types
- *
- * SigLockedSingleOutput: Describes a deposit to a single address which is unlocked via a signature
- * SigLockedDustAllowanceOutput: enables an address to receive dust outputs. It can be consumed as an input like a
- * regular SigLockedSingleOutput
+ * @brief UTXO output types
  *
- * The amount of a SigLockedDustAllowanceOutput must be at least 1 Mi. Apart from this, SigLockedDustAllowanceOutputs
- * are processed identical to SigLockedSingleOutput
+ * Extended output: Describes a deposit to a single address. The output might contain optional feature blocks and
+ *                  native tokens.
+ * Alias output: Describes an alias account in the ledger.
+ * Foundry output: Describes a foundry that controls supply of native tokens.
+ * NFT output: Describes a unique, non-fungible token deposit to a single address.
  *
  */
 typedef enum {
-  OUTPUT_SINGLE_OUTPUT = 0,  ///< SigLockedSingleOutput
-  OUTPUT_DUST_ALLOWANCE      ///< SigLockedDustAllowanceOutput
-} output_type_t;
+  OUTPUT_EXTENDED = 3,  ///< Extended output
+  OUTPUT_ALIAS = 4,     ///< Alias output
+  OUTPUT_FOUNDRY = 5,   ///< Foundry output
+  OUTPUT_NFT = 6        ///< NFT output
+} utxo_output_type_t;
```

Comment:
why not list all output types?
```suggestion
  OUTPUT_SINGLE_OUTPUT = 0, ///< SigLockedSingleOutput, deprecated
  OUTPUT_DUST_ALLOWANCE = 1, ///< SigLockedDustAllowanceOutput, deprecated
  OUTPUT_TREASURY = 2, ///< Treasury, not supported in this library
  OUTPUT_EXTENDED = 3,  ///< Extended output
  OUTPUT_ALIAS = 4,     ///< Alias output
  OUTPUT_FOUNDRY = 5,   ///< Foundry output
  OUTPUT_NFT = 6        ///< NFT output
} utxo_output_type_t;
```

Reply:
All output types will be listed including deprecated and unsupported ones.

Reply:
What is a purpose of a Treasury output? I need it for a description.